### DISCUSSION TITLE
wix usage

### DISCUSSION THREAD

**Discussion by MaximeTuab** [0xfe/vexflow#1512] [General]
Hi! 
Anybody succeeded using vexflow inside a Wix website?
Thank you all!
M.

**Comment by mscuthbert** [comment]
I haven't but it appears you can integrate your own Javascript:

https://a1webstats.com/knowledge-centre/insights/how-to-add-javascript-to-wix-websites/

If you want to quickly render music notation without scripting, I have a package called music21j that uses Vexflow which allows simple notation to be rendered inside any DIV tag so long as it has a certain class on it.  But you might already have the Vexflow skills to do that.
